spec.html

```diff
@@ -29698,6 +29698,11 @@ <h1>Int32Array ( . . . )</h1>
       <p>See <emu-xref href="#sec-typedarray-constructors"></emu-xref>.</p>
     </emu-clause>
 
+    <emu-clause id="constructor-properties-of-the-global-object-iterator">
```

Comment:
```suggestion
    <emu-clause id="sec-constructor-properties-of-the-global-object-iterator">
```

Reply:
I noticed this, but I'd really rather not be so strict about it. If we really want to be strict about it, we should build it into ecmarkup.

Reply:
Thanks, not sure why I missed that. Fixed :)